Add sections for the continuation API bundle to the HOWTO (#574)
fedidcg/FedCM: explorations/HOWTO-chrome.md

  }
}
```

### Continuation API

> This API lets the IdP request that the login-to-RP flow should continue

Reviewer: ```suggestion
This API lets the IdP request that the authorization flow should continue
```

in a popup window that is controlled by the IdP. This can be used to request
additional permission, to ask a user to confirm their account details, or
for a variety of other use cases.

To use this feature:
* Enable the experimental feature `FedCmAuthz` in chrome://flags
* Return a "continue_on" field with a URL instead of a token
  from the ID assertion endpoint. For example:
  ```js
  {
    "continue_on": "https://idp.example/finish_login?account_id=123"
  }
  ```

> * When the login flow finishes, call `IdentityProvider.resolve` to close the

Reviewer: ```suggestion
* When the authorization flow finishes, call `IdentityProvider.resolve` to close the
```

  popup and provide the token that will be passed to the RP:
  ```js
    IdentityProvider.resolve("this is the token");
  ```
* If the account ID has changed (for example, if the popup provided a "Switch
  User" function), you can specify it in a second parameter:
  ```js
    IdentityProvider.resolve("this is the token", {accountId: "123"});
  ```
* If the user cancels the login flow, call `IdentityProvider.close` to close
  the popup and reject the promise that was returned from `navigator.credentials.get`:
  ```js
    IdentityProvider.close();
  ```

### Parameters API
This feature lets RPs specify additional key/value pairs that will get sent
to the ID assertion endpoint.

To use this feature:
* Enable the experimental feature `FedCmAuthz` in chrome://flags
* Add a `params` field to the `navigator.credentials.get` call:
  ```js
    navigator.credentials.get({
        identity: {
          providers: [{
            configURL: "https://idp.example/config.json",
            clientId: "123",
            nonce: nonce,
            params: {

>               "key": "value"

Reviewer: ```suggestion
              "key": "value",
              "anything_goes": "yes",
              "really": "yes",
              "scopes": "calendar.readonly",
              "dpop": "something",
              "moar": "sure",
```

Just expanding with a few more examples.

            }
          }],
        }
    });
  ```
* These key/value pairs will be sent as-is in the ID assertion request:

>   `key=value&account_id=123&...`

Reviewer: ```suggestion
  `key=value&account_id=123&...`
```

OTs are ok to be backwards incompatible, but if we could prefix these with `param-` we could avoid a backwards incompatibility problem later.

Author: the suggested change looks identical to the existing text to me?

Let me keep this as-is to match what is implemented. When we change the code I'll update this howto.

> ### Scaling .well-known

Reviewer: ```suggestion
### Multiple configURLs
```

Maybe we should call it this rather than "scaling well-known"?
